_extensions/posit-docs/theme.scss

```
@@ -3,40 +3,10 @@
 // import font
 @import url("https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap");
 
-/*-- scss:variables --*/
+// import shared colors
+@import '_posit-colors';
 
-// scss-docs-start color-variables
-$posit-blue: #447099;
-$posit-light-blue-1: #d1dbe5;
-$posit-light-blue-2: #a2b8cb;
-$posit-light-blue-3: #7494b1;
-$posit-dark-blue-1: #305775;
-$posit-dark-blue-2: #213d4f;
-$posit-orange: #ee6331;
-$posit-light-orange-1: #edccbf;
-$posit-light-orange-2: #eba38c;
-$posit-dark-orange-1: #ab4d26;
-$posit-dark-orange-2: #80361c;
-$posit-dark-orange-3: #451f12;
-$posit-gray: #404041;
-$posit-light-gray-1: #c2c2c4;
-$posit-light-gray-2: #949494;
-$posit-dark-gray-1: #333333;
-$posit-teal: #419599;
-$posit-light-teal-1: #c2d9d9;
-$posit-light-teal-2: #94bdbf;
-$posit-light-teal-3: #70a3a6;
-$posit-dark-teal-1: #297075;
-$posit-dark-teal-2: #1f4f4f;
-$posit-dark-teal-3: #122b2b;
-$posit-green: #72994e;
-$posit-burgundy: #9a4665;
-$posit-light-burgundy-1: #d9c4cc;
-$posit-light-burgundy-2: #bf96a3;
-$posit-light-burgundy-3: #a67380;
-$posit-dark-burgundy-1: #78384f;
-$posit-dark-burgundy-2: #542938;
-$posit-dark-burgundy-3: #2e171f;
+/*-- scss:variables --*/
 $primary: $posit-teal;
```

Just spoke about this - sorry about that

```suggestion
$primary: $posit-blue;
```